db/models.js

```diff
@@ -37,3 +33,25 @@ exports.fetchArticleById = (article_id) => {
       return article.rows[0];
     });
 };
+
+exports.fetchArticleComments = (article_id, sort_by = "created_at") => {
+  const sortableColumns = ["created_at"];
+
+  if (!sortableColumns.includes(sort_by)) {
+    return Promise.reject({ status: 400, message: "Invalid sort_by." });
+  }
+  return db
+    .query(
+      `SELECT * FROM comments WHERE article_id = $1 ORDER BY ${sort_by} DESC`,
+      [article_id]
+    )
+    .then((comments) => {
+      if (comments.rows.length === 0) {
+        return Promise.reject({
+          message:
+            "This article hasn't received any comments or doesn't exist.",
```

Comment:
This is not an error. When an article is first posted, it won't have any comments and this is not an error.